change/@splunk-otel-d7339ba6-6f53-4a96-95b4-9412de9b319e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "chore: calculate test coverage accurately",
+  "packageName": "@splunk/otel",
+  "email": "Rauno56@gmail.com",
+  "dependentChangeType": "none"
```

Comment:
should this be part of the changelog?

Reply:
Dunno, but we can reconsider this while making the release I assume.
cc @jtmalinowski